markdown.dtx

```diff
@@ -24070,873 +24362,2812 @@ parsers.spnl                   = parsers.optionalspace
-% \end{tikzpicture}
-% \caption{A pushdown automaton that recognizes \TeX{} comments}
-% \label{fig:commented_line}
-% \end{minipage}
-% \end{figure}
+%   \par
 % \begin{markdown}
 %
-% The \luamdef{parsers.commented_line}`^1` parser recognizes the regular
-% language of \TeX{} comments, see an equivalent finite automaton in Figure
-% <#fig:commented_line>.
+%#### Parsers Used for Indentation (local)
 %
 % \end{markdown}
 %  \begin{macrocode}
-parsers.commented_line_letter  = parsers.linechar
-                               + parsers.newline
-                               - parsers.backslash
-                               - parsers.percent
-parsers.commented_line         = Cg(Cc(""), "backslashes")
-                               * ((#(parsers.commented_line_letter
-                                    - parsers.newline)
-                                  * Cb("backslashes")
-                                  * Cs(parsers.commented_line_letter
-                                    - parsers.newline)^1  -- initial
-                                  * Cg(Cc(""), "backslashes"))
-                                 + #(parsers.backslash * parsers.backslash)
-                                 * Cg((parsers.backslash  -- even backslash
-                                      * parsers.backslash)^1, "backslashes")
-                                 + (parsers.backslash
-                                   * (#parsers.percent
-                                     * Cb("backslashes")
-                                     / function(backslashes)
-                                       return string.rep("\\", #backslashes / 2)
-                                     end
-                                     * C(parsers.percent)
-                                     + #parsers.commented_line_letter
-                                     * Cb("backslashes")
-                                     * Cc("\\")
-                                     * C(parsers.commented_line_letter))
-                                   * Cg(Cc(""), "backslashes")))^0
-                               * (#parsers.percent
-                                 * Cb("backslashes")
-                                 / function(backslashes)
-                                   return string.rep("\\", #backslashes / 2)
-                                 end
-                                 * ((parsers.percent  -- comment
-                                    * parsers.line
-                                    * #parsers.blankline) -- blank line
-                                   / "\n"
-                                   + parsers.percent  -- comment
-                                   * parsers.line
-                                   * parsers.optionalspace)  -- leading tabs and spaces
-                                 + #(parsers.newline)
-                                 * Cb("backslashes")
-                                 * C(parsers.newline))
```

Comment:
@lostenderman: Where did these lines go? I am aware that we removed HTML block comments, but these patterns (and diagram) are related to the unrelated `texComments` Lua option.

Reply:
Please, nevermind, I can see them now.